runtime/testing-hydradx/src/lib.rs

```diff
@@ -722,7 +722,7 @@ parameter_types! {
 		// Alice
 		hex!["d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"].into(),
 		// Bob
-		hex!["8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"].into(),
+		// hex!["8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"].into(),
```

Comment:
commented this out for the test to verify that Bob gets paid.
Why do we not pay testing collators?

Reply:
i guess this is just setup this way to test excluded collators set working correctly, we are not paying GC collators as well

Reply:
So remove this entry so we have one excluded collator (Alice) and one that gets rewards (Bob)?

Reply:
sure, that would be nice